src/control/server/ctl_ranks_rpc.go

```diff
@@ -191,7 +191,7 @@ func (svc *ControlService) StopRanks(ctx context.Context, req *ctlpb.RanksReq) (
 
 	// ignore poll results as we gather state immediately after
 	if err := pollInstanceState(ctx, instances, func(e Engine) bool { return !e.IsStarted() }); err != nil {
-		return nil, errors.Wrap(err, "CtlSvc.StopRanks pollInstanceState")
+		svc.log.Error(errors.Wrap(err, "CtlSvc.StopRanks pollInstanceState").Error())
```

Comment:
Are these actually errors? If not, then they shouldn't be logged as such because they're going to potentially cause concern when there shouldn't be. Another way of putting it is: Who are these log entries for? If the admin sees them in the log, what should they do about them? If they're not for the admin, then they should be logged at debug or discarded.

Reply:
These are server side errors that should be logged because the state didn't change as intended within the allotted context deadline (or because the context was cancelled). Don't return the error here because we want to populate the response so that the dmg caller receives meaningful reason for failure.

Reply:
OK, I had to go look at how `pollInstanceState()` works now to understand what's going on here. So now the only way for that function to return an error is if the context is canceled, right? Looking at the way the function is used now, there is no longer a server-side timeout, so the only way those contexts will be canceled is if the client cancels them or if the server is shut down while they're still running, as far as I can see.

In either of those scenarios, then, I'm pretty sure what would be logged would look something like this:
```
ERROR 12:31:01: CtlSvc.StopRanks pollInstanceState: context canceled
```

I'm not really sure how useful that is. Furthermore, if the context really was canceled, shouldn't the RPC handler stop processing right away and return? I'm willing to believe that I'm not understanding this correctly, but at the moment it seems as if this PR is swallowing legitimate errors, which could lead to confusing behavior at best.